lib/inspect_subpackages.c

```diff
@@ -38,13 +49,13 @@ bool inspect_subpackages(struct rpminspect *ri)
     TAILQ_FOREACH(peer, ri->peers, items) {
         if (peer->before_hdr) {
             entry = calloc(1, sizeof(*entry));
-            xasprintf(&entry->data, "%s %s", headerGetString(peer->before_hdr, RPMTAG_NAME), headerGetString(peer->before_hdr, RPMTAG_ARCH));
```

Comment:
This line could probably stay by just replacing the headerGetString for RPMTAG_ARCH with a call to get_rpm_header_arch().  The format_subpackage_name_arch() function could be removed.